src/components/Tab/style.scss

```diff
@@ -6,19 +6,18 @@
   flex-direction: row;
   align-items: center;
   padding: 0 24px;
-  height: 49px;
   border: none;
   background: none;
   outline: none;
   color: $font-primary-color;
   cursor: pointer;
   transition: color 0.25s;
-  margin: 0 8px;
-  font-size: 15px;
+  margin: 0 8px 11px 8px;
+  font-size: 14px;
 
   &__description {
     display: inline-block;
-    color: #000;
+    color: #8e949a;
```

Comment:
After discussion with @kamilmateusiak he told me to put it directly as there are some changes planned for colors variables anyway